src/codeu/chat/client/core/View.java

```diff
@@ -136,4 +136,23 @@ public View(ConnectionSource source) {
 
     return messages;
   }
+
+  public ServerInfo getInfo() {
+    try (final Connection connection = source.connection()) {
+      Serializers.INTEGER.write(connection.out(), NetworkCode.SERVER_INFO_REQUEST);
+      if (Serializers.INTEGER.read(connection.in()) == NetworkCode.SERVER_INFO_RESPONSE) {
+        final Time startTime = Time.SERIALIZER.read(connection.in());
+        return new ServerInfo(startTime);
+      } else {
+      // Communicate this error - the server did not respond with the type of
```

Comment:
log the error here 
`LOG.error(ex, "Exception during call on server.");`
ditto below